uber/templates/jobs/signups.html

```diff
@@ -81,7 +81,7 @@
                     </form>
                     <td id="rating{{ shift.id }}"></td>
                     {% if shift.worked == SHIFT_WORKED %}
```

Comment:
IDs need to be quoted since they're uuids now and not integers.  I thought I had already caught all of the places where this came up, but apparently not!